src/payment-request.ts

```diff
@@ -1,8 +1,22 @@
 import { gql, GraphQLClient } from "graphql-request";
 import { getDetailedSessionById } from "./session";
+import { getLatestFlowGraph } from "./flow";
 import keyPathAccessor from "lodash.property";
 import setByKeyPath from "lodash.set";
-import type { PaymentRequest, Session, KeyPath, Value } from "./types";
+import { ComponentType } from "./types";
+import type {
+  PaymentRequest,
+  Session,
+  KeyPath,
+  Value,
+  Node,
+  FlowGraph,
+} from "./types";
+
+type ValidatedSessionData = {
+  sessionPreviewData: PaymentRequest["sessionPreviewData"];
+  amount: number;
```

Comment:
nit: i'd prefer to standardise around the language of `fee` as that feels more specific to the process of paying/requesting payments & better aligns to existing LDC schemas (eg `application.fee.payable`), whereas `amount` reads as broader to me and I think could be more easily confused with other things in content like number of homes on a site, amount of windows being replaced, etc!